### DISCUSSION TITLE
How do I use the rights file?

### DISCUSSION THREAD

**Discussion by Janik-Haag** [Kozea/Radicale#1378] [Q&A general]
I'm trying to have a calendar that everyone can read but only I can write to.

```ini
[calendars]
user: .*
collection: myUser/publiclySharedCalendar
permissions: r
```
shouldn't this allow anyone to read the calendar? 
My auth method is `htpasswd`

**Comment by BLACK4585** [comment]
//bump
Also searching for this for a while now since I don't want to use symlinks. The documentation regarding the arguments which can be given in this file is very weak.
